docs/components/ButtonGroup/Mobile.PrimaryAction.stories.tsx

```diff
@@ -6,6 +6,7 @@ export default {
   title: "Components/Actions/ButtonGroup/Mobile/ButtonGroup.PrimaryAction",
```

Comment:
Are we aware of the errors in these ButtonGroup examples?

![Screenshot 2023-07-19 at 11 03 53 AM](https://github.com/GetJobber/atlantis/assets/104797704/060906b8-1690-4e58-9d74-721582f7f96e)

Reply:
yup. that's what the disclaimer is for